.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing to PSReadLine
+
+We welcome and appreciate contributions from the community.
+There are many ways to become involved with PSReadLine, including
+
+- Filing issues;
```

Comment:
Remote the semicolons and final period since this is a bulleted list

Reply:
Will do.